3.0/Dockerfile

```diff
@@ -1,10 +1,12 @@
 # vim:set ft=dockerfile:
 
-FROM cimg/base:2021.10
+# Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
+
+FROM cimg/base:2022.04
 
 LABEL maintainer="Community & Partner Engineering Team <community-partner@circleci.com>"
 
-ENV RUBY_VERSION=3.0.3 \
+ENV RUBY_VERSION=3.0.1 \
```

Comment:
Is this a typo?
The current version for 3.0.x branch is 3.0.4

Reply:
Correct, this was a mistake. Good catch. A fix is already on it's way out.